lib/react_client.dart

```diff
@@ -229,6 +228,24 @@ Map<String, dynamic> _unjsifyContext(InteropContextValue interopContext) {
   });
 }
 
+dynamic _jsifyNewContext(dynamic context) {
+  if (context is Map) {
+    return new JsBackedMap.from(context).jsObject;
+  }
+  return context;
+}
+
+@JS()
+external get JsBackedMapSymbol;
+
+dynamic _unjsifyNewContext(dynamic interopContext) {
+  if (interopContext is JsMap) {
+    return JsBackedMap.copyToDart(interopContext);
```

Comment:
This could result in type errors when the Map is converted back to Dart, since it's always going to be a `Map<dynamic, dynamic>`.

Can we remove the special case for maps?